cookbook/profiler/data_collector.rst

```diff
@@ -203,7 +203,7 @@ must also define additional blocks:
             </div>
         {% endset %}
 
-        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig') }}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', {'link': true}) }}
```

Comment:
Or shouldn't we better set this to `false`like it is done in the example before? Otherwise it might look strange for the reader that we changed the value.